### DISCUSSION TITLE
can't search anything after spotify integration

### DISCUSSION THREAD

**Discussion by 7549brijesh** [Moosync/moosync-app#301] [General]
after adding spotify client id and secret id nothing is fetching not even single song(like a blank canvas)
am I missing something??

**Comment by Ovenoboyo** [comment]
Hey, once you input the client ID and secret, you'll need to login to your Spotify account. You can do that by pressing the person icon at the top right of the music player window.

Once you're logged in, the playlists in your Spotify library should show up under the playlists tab and you'll also be able to view recommendations (in the explore tab) and search Spotify songs
